### PR TITLE
Update protectli building manuals

### PR DESCRIPTION
**Pull request by macpijan**

### CODE REVIEW

**Comment by miczyg1** [comment]
@macpijan https://github.com/Dasharo/docs/pull/567

File: docs/variants/protectli_vp2410/building-manual.md [hunk 1]
```diff
@@ -14,25 +14,49 @@ VP2410.
...
+    git clone https://github.com/Dasharo/coreboot
     ```
 
-2. Check out the desired version e.g. `v1.0.10`:
+1. Checkout the desired version, e.g. `v1.0.15`:
 
     ```bash
     cd coreboot
-    git checkout protectli_vault_glk_v1.0.10
+    git checkout protectli_vault_glk_v1.0.15
```
[the remark this answers is not in the document]
Comment (macpijan) [reply docs/variants/protectli_vp2410/building-manual.md:31]: https://github.com/Dasharo/edk2/commit/90364638844f6bd9a257f489a9ec009d88d85922 says: `https://github.com/Dasharo/edk2/commit/90364638844f6bd9a257f489a9ec009d88d85922`

In log I can see: 

```
    Fetching new commits from the Tianocore repo
Fetching origin
Fetching upstream
    Checking out Tianocore revision 90364638844f6bd9a257f489a9ec009d88d85922
fatal: reference is not a tree: 90364638844f6bd9a257f489a9ec009d88d85922
```

The tiaocore repo is checked out to: `19bf14b42b5705d9737cb4cba273fc80534de740` in sources.

[the remark this answers is not in the document]
Comment (miczyg1) [reply docs/variants/protectli_vp2410/building-manual.md:31]: I have created protectli_uefipayload branch from the commit 90364638844f6bd9a257f489a9ec009d88d85922 and now it checks out the commit successfully

**Review by Pokisiekk — APPROVED** [review]
I tested the documentation for the Vp2420 and built the firmware without any problems.

**Pull request merged**